src/Manager/Environment/environmentReducer.ts

```diff
@@ -167,6 +167,11 @@ export const progressLabel = (env: Environment) =>
           }`
         : '';
 
+export const isUnsupportedEnvironment = (environmentVersion: string) => {
```

Comment:
Maybe very nitpicky, but imo it would be better to have it as `isSupportedEnvironment`
Makes it clearer and easier to understand (plus usually the affirmative case is more common in our codebase)

Reply:
Wanted to implement your comment and just recalled :) first intention was to have it as `isSupportedEnvironment`, for sure clearer option, but in this case I have to use lt() function, should be lt(environmentVersion, 'v3.0.0'); but it didn't work well for the version `3.0.0-preview1`, so ended up with gte() and `isUnsupportedEnvironment` variable name.
Could have 
`const isSupportedEnvironment = ! isUnsupportedEnvironment` but looks like an overkill :)